packages/errors/src/errors.ts

```diff
@@ -570,7 +570,6 @@ export const AllCypressErrors = {
         This can happen for a number of different reasons:
 
         - You wrote an endless loop and you must fix your own code
-        - There is a memory leak in Cypress (unlikely but possible)
```

Comment:
Removing this to not first plant the idea that the user isn't at fault. They may not be at fault, but we'd like more investigation before they reach that conclusion.